### ISSUE TITLE
Set up Ingress on Minikube with the NGINX Ingress Controller- Minikube runs version of k8s incompatible with ingress sample

### ISSUE THREAD

**Issue by sanguis** [kubernetes/website#29606]
In the guid on 
https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/
The version of `minikube` spun up is version 1.17 while the example ingress is agains the api provided by minikube 1.19 and up.
The ingress that works is.

```
apiVersion: networking.k8s.io/v1beta1
kind: Ingress
metadata:
  name: example-ingress
  annotations:
    nginx.ingress.kubernetes.io/rewrite-target: /$1
spec:
  rules:
    - host: hello-world.info
      http:
        paths:
          - path: /
            backend:
              serviceName: web
              servicePort: 8080
```

I know it would be better to get a modern minikube going but this is a fix for now

**Comment by niteshseram** [comment]
Similar issue #29204

**Comment by RA489** [comment]
the latest release of minikube solve this problem https://github.com/kubernetes/minikube/pull/12089

**Comment by sftim** [comment]
Looks like #29616 would fix this.

**Comment by sftim** [comment]
Looks to be fixed in #29616
/close
